autokeras/auto/auto_model.py

```diff
@@ -96,21 +84,29 @@ def __init__(self,
         self._hypermodels = []
         self._hypermodel_to_id = {}
         self._build_network()
+        self.tuner = tuner.SequentialRandomSearch(
+            self,
+            objective=self._get_metrics())
 
-    def build(self, hp):
+    def build(self, hp, **kwargs):
```

Comment:
Where are `**kwargs` used?

If another piece of code need to programatically call `build`, how can it know what arguments to pass?

Reply:
This is removed since we are not using the **kwargs in the build function.